tiled/client/constructors.py

```diff
@@ -268,7 +268,7 @@ def from_profile(name, structure_clients=None, **kwargs):
                 f"ValidationError while parsing configuration file {filepath}: {msg}"
             ) from err
         context = Context.from_app(
-            build_app_from_config(config, source_filepath=filepath), **merged
```

Comment:
where do the things that used to come in via `**merged` get threaded through instead now?

Reply:
It's _just_ below the fold shown by GitHub, Line 275.

Reply:
That is is the "else" case, in this branch it appear to be silently ignored.